sutraani/sutrartha_english.txt

```diff
@@ -3971,8 +3971,8 @@
     "84055": "A झल्  letter is converted to a चर्  letter when followed by a खर्  letter.",
     "84056": "A झल्  letter is converted to a चर्  letter optionally when it is followed by an अवसान.",
     "84057": "The अण् letter of a non-प्रगृह्य word becomes अनुनासिक optionally when it is followed by an अवसान. ",
-    "84058": "An अनुस्वार is converted to a परसवर्ण in presence of a  यय् letter.",
-    "84059": "An अनुस्वार occurring at end of a पद is converted to an अनुस्वार optionally.",
+    "84058": "An अनुस्वार when followed by a यय् वर्ण is converted to a परसवर्ण optionally.",
+    "84059": "An अनुस्वार occurring at end of a पद when follwed by a यय् वर्ण is converted to an अनुस्वार optionally.",
```

Comment:
Both of these have mistakes, but I am merging them and I will fix the mistakes on my side, That is faster than to-and-fro messaging.